app/javascript/react/components/Modals/SessionDetailsModal/SessionInfo/DesktopHeader.tsx

```diff
@@ -28,6 +29,12 @@ interface DesktopHeaderProps {
   fixedSessionTypeSelected: boolean;
 }
 
+type CustomPopupProps = {
+  children:
+    | React.ReactNode
+    | ((close: () => void, isOpen: boolean) => React.ReactNode);
+};
+
 const DesktopHeader: React.FC<DesktopHeaderProps> = ({
```

Comment:
LOL, We use the same name for the component used in Navbar 😅 I propose changing this name to sth connected with a modal like `ModalDesktopHeader` WDYT?